lib/semian/mysql2.rb

```diff
@@ -0,0 +1,52 @@
+require 'semian'
+require 'mysql2'
+
+class Mysql2::SemianError < Mysql2::Error
+  def initialize(semian_identifier, *args)
+    super(*args)
+    @semian_identifier = semian_identifier
+  end
+
+  def to_s
+    "[#{@semian_identifier}] #{super}"
+  end
+end
+
+module Semian
+  module Mysql2
+    def semian_identifier
+      @semian_identifier ||= begin
+        name = query_options[:semian] && query_options[:semian][:name]
```

Comment:
Hum... I don't now. `Semian.register` have default values. If we want these to be always explicitly defined, then we might as well remove the default values no?